vault/external_tests/audit/audit_filtering_test.go

```diff
@@ -289,6 +289,39 @@ func TestAuditFilteringFallbackDevice(t *testing.T) {
 	require.Equal(t, 5, numberOfEntries)
 }
 
+// TestAuditFilteringFilterForUnsupportedField validates that the audit device
+// 'filter' option fails when the filter expression selector references an
+// unsupported field and that the error prevents an audit device from created.
+func TestAuditFilteringFilterForUnsupportedField(t *testing.T) {
+	t.Parallel()
+	cluster := minimal.NewTestSoloCluster(t, nil)
+	client, err := cluster.Cores[0].Client.Clone()
+	require.NoError(t, err)
+	client.SetToken(cluster.RootToken)
+
+	tempDir := t.TempDir()
+	filteredLogFile, err := os.CreateTemp(tempDir, "")
+	filteredDevicePath := "filtered"
+	filteredDeviceData := map[string]any{
```

Comment:
I'm not sure if this needs to be in a new test or could be added to this one, but we should test that it's also not created with `skip_test=true`. I'm not sure what the use case for that is normally, but hopefully it won't get around this validation.

Reply:
It for sure never hurts to test different settings interacting with each other. Added some logic to this test to test the same behaviour with `skip_test=true`. Thanks for the suggestion.